README.md

```diff
@@ -1,8 +1,67 @@
-
 # oas-discovery-proxy
 
-This is a placeholder README.md for a new repository
+Proxy for the OAS Discovery API Service. Serves as the OAS Discovery API in a
+non-prod environment. Forwards calls to the real OAS Discovery API via HODS Proxy
+and Scrubbing Centre.
+
+In environments without a real OAS Discovery API then this proxy forwards to the OAS Discovery API stub,
+oas-discovery-stubs.
+
+## Requirements
+
+This service is written in [Scala](http://www.scala-lang.org/) and [Play](http://playframework.com/), so needs at least a [JRE] to run.
+
+## Dependencies
+Beyond the typical HMRC Digital platform dependencies this service relies on:
+- OAS Discovery API Service
+
+The full set of dependencies can be started using Service Manager and the group API_HUB_ALL.
+
+You can view service dependencies using the Tax Catalogue's Service Relationships
+section here:
+https://catalogue.tax.service.gov.uk/service/identity-management-service-proxy
+
+### OAS Discovery API
+All requests to this service starting with are forwarded to IDMS. 
```

Comment:
starting with `/oas-discovery-proxy/internal` are forwarded to the `OAS Discovery API`

Reply:
Ok